index.html

```diff
@@ -691,33 +691,17 @@ <h4>Proof Configuration (ecdsa-rdfc-2019)</h4>
 
           <ol class="algorithm">
             <li>
-Let <var>proofConfig</var> be an empty object.
+Let <var>proofConfig</var> be a clone of the <var>options</var> object.
```

Comment:
I avoid cloning the options object because it could contain information that has nothing to do with this algorithm. At worst, it could get the implementer to implement something that allows an injection attack (from the network) of some kind. All the algorithms are written such that injection attacks are avoided.

Reply:
The cloning is what enables extensibility and makes this PR solve the issue -- without it, the issue would not be solved. I think "preventing injection attacks from the network" here would be done at the wrong layer and there is already a presumption that inputs will always be appropriately sanitized when passed to a signing library. Otherwise, an attacker could ask for anything to be signed; it doesn't matter whether the proof options are "built-ins" or extensible ones.

So, -1 to changing this, I think it's the right approach for solving the issue.